src/features/prompts/components/prompts-table.tsx

```diff
@@ -106,12 +76,12 @@ export function PromptTable(props: { projectId: string }) {
       accessorKey: "action",
```

Comment:
We discussed this differently, but i think it does not make sense to have this button here anymore.